lib/matplotlib/cbook/__init__.py

```diff
@@ -989,7 +989,12 @@ def _combine_masks(*args):
         else:
             if isinstance(x, np.ma.MaskedArray) and x.ndim > 1:
                 raise ValueError("Masked arrays must be 1-D")
-            x = np.asanyarray(x)
+            try:
+                x = np.asanyarray(x)
+            except np.VisibleDeprecationWarning:
```

Comment:
So, what happens after the deprecation period?

Reply:
I assume it'll be a `ValueError`, but it wasn't clear in the NEP.